migrator/config-loader.lua

```diff
@@ -0,0 +1,60 @@
+local fio = require('fio')
+local checks = require('checks')
+local log = require('log')
+
+local Loader = {}
+Loader.__index = Loader
+
+local function __must_sort(m)
+    table.sort(m, function(a, b) return a.name < b.name end)
+end
+
+local function assert_migration(migration)
+    checks({
+        name = 'string',
+        up = 'function'
+    })
+    return migration
+end
+
+function Loader:list()
+    local ca = require("cartridge.confapplier")
+    local cfg = ca.get_active_config():get_plaintext()
+    if cfg == nil then
+        return {}
+    end
+
+    local result = {}
+
+    for k, v in pairs(cfg) do
+        if fio.dirname(k) == self.dir_name then
+            local migration, err = dostring(v)
+            if migration ~= nil then
+                migration.name = fio.basename(k)
+                assert_migration(migration)
+                table.insert(result, migration)
+            else
+                log.warn('Cannot load %s: %s', v, err)
+            end
+        end
+    end
+
+    __must_sort(result)
+
+    return result
+end
+
+local function new(dir_name)
```

Comment:
Is it really config section name? I'd suggest to rename it then

Reply:
removed option completely. "place you migrations code in this section" only